EIPS/eip-1.md

```diff
@@ -135,6 +135,8 @@ Each EIP must begin with an [RFC 822](https://www.ietf.org/rfc/rfc822.txt) style
 
 `created`: *Date the EIP was created on*
 
+`finalized`: *Date the EIP's status turns FINAL* (Optional field, only needed when turning FINAL)
```

Comment:
How this:

```suggestion
`finalized`: *Date the EIP was finalized* (Optional field, only needed when status is `Final` or `Withdrawn`)
```

Reply:
I hesitate to use "finalized" for the EIPs of withdrawn, I wonder if it shall have a separate date or just a more general date of "last updated"

Reply:
In EIP-1's description of `Withdrawn`:

> This state has finality and can no longer be resurrected using this EIP number.

I think `finalized` applies to both, since they have "finality".

Reply:
That makes sense